2016/playbook/session_3/experiments/improve_social_behavior_of_hackers_with_card_games.md

```diff
@@ -0,0 +1,19 @@
+# Running card games during lunch
+
+## Implementation
+
+-Run a card game during lunch which initiates conversation and increases hacker productivity after lunch
+
+## Observations
+
+- Ran 3 rounds of Mafia during lunch
+
```

Comment:
Markdown lists should not have spaces between each bullet. Check out https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet for reference